lib/src/components/_stacks-tables.less

```diff
@@ -197,17 +151,172 @@
     }
 }
 
-//  $$  Footer: Large Row
-//  ------------------------------------------------------------------------
-.s-table--totals {
-    th,
-    td {
-        padding-top: @su12;
-        font-size: @fs-subheading;
-        font-weight: bold;
 
-        &:hover {
-            background-color: transparent;
+//  ============================================================================
+//  $ LAYOUT VARIATIONS
+//  ----------------------------------------------------------------------------
+.s-table {
+    //  $$  HOVER
+    //  ------------------------------------------------------------------------
+    &.s-table__hover tbody tr:hover {
+        background-color: @black-025;
+    }
+
+    //  $$  SPACING
+    //  ------------------------------------------------------------------------
+    //  --  SMALL
+    &.s-table__sm {
+        th,
+        td {
+            padding: @su4;
         }
     }
+    //  --  MEDIUM
+    &.s-table__md {
+        th,
+        td {
+            padding: @su12;
+        }
+    }
+    //  --  LARGE
+    &.s-table__lg {
+        th,
+        td {
+            padding: @su16;
+        }
+    }
+
+    //  $$  NO BORDERS
+    //  ------------------------------------------------------------------------
+    //  --  X & Y Axis
+    &.s-table__b0 {
+        th,
+        td,
+        tr:last-of-type th,
+        tr:last-of-type td {
+            border-color: transparent;
+        }
+
+        //  This makes the border transparent, so we need to use whitespace
+        //  to achieve the same effect a 2px gray border achieves.
+        tbody + tbody {
+            border-color: transparent;
+            border-width: @su12;
+        }
+    }
+    //  --  X-AXIS
+    //      Clear out the y-axis colors
+    &.s-table__bx0 {
+        th,
+        td {
+            border-left-color: transparent;
+            border-right-color: transparent;
+        }
+        thead th {
+            border-top-color: transparent;
+            border-bottom-color: @black-400;
+        }
+        tbody tr {
+            &:first-of-type th,
+            &:first-of-type td {
+                border-top-color: transparent;
+            }
+            &:last-of-type th,
+            &:last-of-type td {
+                border-bottom-color: transparent;
+            }
+        }
+        tfoot th,
+        tfoot td {
+            border-bottom-color: transparent;
+        }
+    }
+
+    //  --  Y-AXIS
+    &.s-table__by0 {
```

Comment:
How necessary are these classes? These tables, especially this one, are really awkward to me, aesthetically.

Reply:
Do you think they are all unnecessary or just borderless and no y borders?

Reply:
We need borderless and borderless-x IMO. Borderless-y I'm struggling to come up with a use case within the product.